update.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -e
+
+msg () {
+	1>&2 echo $*
+}
+
+# check that we are in a prototype
+if ! grep -q govuk-prototype-kit package.json 2> /dev/null; then
+	msg 'ERROR you must run update.sh in a folder containing a GOV.UK Prototype Kit installation'
+	msg 'Exiting'
+	exit 1
+fi
+
+# make the update folder
+mkdir -p update
+echo '*' > update/.gitignore
+
+# download the latest Prototype Kit zip file
+if [ ! -f govuk-prototype-kit.zip ]; then
+	msg 'Downloading latest version of GOV.UK Prototye Kit...'
+	curl -Lo govuk-prototype-kit.zip https://govuk-prototype-kit.herokuapp.com/docs/download
+	msg 'Done'
+fi
+
+# unzip the file into the 'update' folder
+if [ ! -d update/govuk-prototype-kit-* ]; then
+	msg -n 'Extracting latest files to update folder...'
+	unzip -q -d update govuk-prototype-kit.zip
+	msg 'Done'
+fi
+
+# copy from the update folder into the current prototype folder
+msg -n 'Updating your prototype files... '
+rsync -r \
+	--log-file=update/update.log \
+	--delete \
+	--exclude=.git/ \
+	--exclude=app/ \
+	--exclude=update/ \
+	--exclude=node_modules/ \
+	--exclude=.env \
+	--exclude=update.sh \
+	--exclude=usage-data-config.json \
+	update/govuk-prototype-kit-*/ ./
+msg 'Done'
+
```

Comment:
can we do step 8: Copy app/assets/sass/patterns from the unzipped folder to your prototype.

Reply:
Yep good idea